book/mvc.md

```diff
@@ -1,31 +1,30 @@
 MVC
 ===
 
-Since you're working with web development, most probably you've heard "MVC" and know that it refers to "Model", "View" and "Controller".
-The "MVC" thing was introduced to explain the principles of building desktop applications: how a user interface is interacting with program itself and the data. Then it was adapted for web development but since web, and especially PHP, is a bit different it didn't fit perfectly.
+If you've been working in web development you've most probably heard of the term "MVC" and know that it refers to "Model", "View", and "Controller". The "MVC" software pattern was introduced in the nineties to explain the principles of building desktop applications. In particular to describe how a user interface interacts with a program and its underlying data. Later it was adapted by web developers and has become a core foundation of many popular web frameworks including PHP frameworks.
```

Comment:
One of the key points lost: `it was adapted for web development but since web, and especially PHP, is a bit different it didn't fit perfectly`.